src/CoreText/CTStringAttributes.cs

```diff
@@ -281,6 +294,12 @@ public CTStringAttributes (NSDictionary dictionary)
 			}
 		}
 
+		[iOS (10,0)][Mac(10,12)]
+		public int? HorizontalInVerticalForms {
+			get {return Adapter.GetInt32Value (Dictionary, CTStringAttributeKey.HorizontalInVerticalForms);}
```

Comment:
Add a test case for make sure there's no `ArgumentNullException` thrown by `GetInt32Value` in old version of iOS where `CTStringAttributeKey.HorizontalInVerticalForms` could be `null`.